mitodl/open-discussions
# MIT Professional Education course import

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #2737

#### What's this PR do?
- Imports courses from https://professional.mit.edu
- Maps topics for the above courses (and Sloan courses) to EdX topics

#### How should this be manually tested?
- Add `MITPE_BASE_URL=https://professional.mit.edu/` to your .env file
- Run `manage.py backpopulate_mitpe_data`
- Go to the course search page, there should be about 80 Professional Education courses there

#### Screenshots
<img width="1130" alt="Screen Shot 2020-04-06 at 3 09 23 PM" src="https://user-images.githubusercontent.com/187676/78597263-33661680-781b-11ea-8a8a-66aa8c5c8083.png">

<img width="503" alt="Screen Shot 2020-04-06 at 3 09 42 PM" src="https://user-images.githubusercontent.com/187676/78597271-37923400-781b-11ea-9903-91af05c4ae03.png">
